nano/node/node.cpp

```diff
@@ -608,6 +608,20 @@ nano::process_return nano::node::process (nano::block const & block_a)
 	return result;
 }
 
+nano::process_return nano::node::process_local (std::shared_ptr<nano::block> block_a)
+{
+	// Add block hash as recently arrived to trigger automatic rebroadcast and election
+	block_arrival.add (block_a->hash ());
+	// Set current time to trigger automatic rebroadcast and election
+	nano::unchecked_info info (block_a, block_a->account (), nano::seconds_since_epoch (), nano::signature_verification::unknown);
+	// Notify block processor to release write lock
+	block_processor.wait_write ();
+	// Process block
+	auto transaction (store.tx_begin_write ());
+	auto result (block_processor.process_one (transaction, info));
+	return result;
+}
```

Comment:
Maybe just `return block_processor.process_one(...)` as the result local isn't used.